### PR TITLE
VIH-8141 added constants for browser names, added samsung as supported browser

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link  ###
https://tools.hmcts.net/jira/browse/VIH-8141

### Change description ###
- added samsung as supported browser

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.spec.ts [hunk 1]
```diff
@@ -97,15 +98,16 @@ describe('DeviceType', () => {
...
+        { browser: browsers.Firefox, expected: true },
+        { browser: browsers.Safari, expected: true },
+        { browser: browsers.Chrome, expected: true },
+        { browser: browsers.MSEdge, expected: true },
+        { browser: browsers.Samsung, expected: true },
+        { browser: browsers.MSEdgeChromium, expected: true },
+        { browser: browsers.Opera, expected: false },
+        { browser: browsers.Brave, expected: false },
+        { browser: browsers.MSIE, expected: false },
+        { browser: browsers.MSIE, expected: false }
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.spec.ts:110]: This was present previously but could we remove this duplication?

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/browser.constants.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const browsers = {
+    Firefox: 'Firefox',
+    Safari: 'Safari',
+    Chrome: 'Chrome',
+    MSEdge: 'MS-Edge',
+    MSEdgeChromium: 'MS-Edge-Chromium',
+    Samsung: 'Samsung',
+    Opera: 'Opera',
+    Brave: 'Brave',
+    MSIE: 'MSIE'
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/browser.constants.ts:10]: Maybe make the key InternetExplorer or MSInternetExplorer just so it's a bit more obvious without having to think.

**Review by FarronF — APPROVED** [review]
Looks good, one minor comment but I'll leave that to you. Let me know if you change it and I'll reapprove

**Review by FarronF — APPROVED** [review]

**Review by robertbennett1998 — APPROVED** [review]
LGTM

**Pull request merged**